service/authService.ts

```diff
@@ -0,0 +1,24 @@
+import type { Login } from "../model/auth";
+import axios from "axios";
+
+export const login = async function (login: Login): Promise<string> 
+{
+    try{
+        const response = await axios.post("http://localhost:8080/api/login", login)
```

Comment:
Hardcoding the url here, will prevent it from working in the cloud. The base URL will need to be passed in using a env variable